src/Altinn.Platform/docker-compose.yml

```diff
@@ -14,6 +14,8 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=http://+:5010
+      - PlatformStorageSettings:ApiEndPoint=http://altinn-platform-storage:5010/api/v1
```

Comment:
Needs to be added to the runtime helm chart as well

Reply:
Can you resolve this when you update the helm chart and complete the pr? @hannekot

Reply:
Helm chart is updated 🔋